package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "peaks.js",
-  "version": "0.10.1",
+  "version": "0.11.0",
```

Comment:
Please don't bump the version number, as merging this commit would automatically trigger publication of the new version to npm. I'll do that when I prepare the next release, which will include some other changes.

Reply:
I took it out